wasm-host/src/main.rs

```diff
@@ -44,6 +44,11 @@ struct Args {
     /// Function to run in the WASM module
     #[arg(long, default_value = "finish")]
     function: String,
+
+    /// Gas cap can be changed for debugging.
+    /// Note that default value is the default used in rippled.
+    #[arg(long, default_value = "100000")]
```

Comment:
The gas cap in rippled is currently 1m, so we should match that here as the default.

Reply:
thanks, updated to 1M